ami/users/api/serializers.py

```
@@ -3,14 +3,15 @@
 
 from ami.main.api.serializers import DefaultSerializer
+from ami.main.models import UserProjectMembership
 
 User = get_user_model()
 
 
 class UserListSerializer(DefaultSerializer):
     class Meta:
         model = User
-        fields = ["id", "name", "details", "image"]
+        fields = ["id", "name", "details", "image", "email"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)